### DISCUSSION TITLE
Q: How to add custom command to bin/console.php?

### DISCUSSION THREAD

**Discussion by vladsf** [eventum/eventum#1215] [Q&A] [answered]
Hi!

How to add a custom command to `bin/console.php`? I copied `src/Console/Command/LdapSyncCommand.php` to another file, 
changed it properly, but `bin/console.php list` does not list it. What could go wrong?

**Comment by glensc** [comment]
not enough details. without details could get anything wrong, as you already felt it with the ExampleExtension, you didn't share enough details about what you did wrong, so I can not analyze what you did wrong.

looking at the crystal ball:

1. does the command work?
2. list command is overridden to show only eventum prefixed commands:
   - src/Console/Command/ListCommand.php
   - use `-a` to show all commands in list output


it be must easier to understand what you are struggling with if you submit a PR and ask there. then could check out the code locally and test.

**Reply by vladsf** [reply]
Sure, here is the sample - #1216

**Comment by vladsf** [comment, marked as answer]
The Q answered here: #1216
